### PR TITLE
Initial version of the Readme page in English

### PR DESCRIPTION
**Pull request by Gaket**
Created a translation of the ReadMe page, we can add a link to it from the main page.

Several questions should be answered:

1. How to word the fact that the repo is primarily in Russian for now, as well as the MindMaps?
2. What to do with the links to pages on Russian? I see 2 possible solutions: 
   a) remove until we get them translated to English.
   b) add [RU] tags, so people will be able to see the Russian version through some Google translator if they want.

I've checked the translation for logical and grammatical correctness, but feel free to ping me if there are some improvements to be done.

Here is the issue for it: https://github.com/tlbootcamp/tlroadmap/issues/117

### CODE REVIEW

File: translations/README-en.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
+![Logotype](../logo.png)
+
+> Team Lead is a ❄️, because in each company they are unique and distinct.
+
+- [Roadmap](#roadmap)
+- [Telegram chat TL Bootcamp (RU)](https://tlinks.run/tlbootcamp)
+
+# What is it about?
+
+The concepts of "Team Lead", "Tech Lead", "Tech Manager" and others are very stretchy. Companies solve different tasks, these roles have different responsibilities. Besides, different leadership styles, differences in personal skills, and much more are put on the picture. But isn't that what happens to the developers? Different companies, different tasks, again, different nature of tasks and responsibilities.
```
Comment (php-coder) [inline translations/README-en.md]:
```suggestion
The concepts of "Team Lead", "Tech Lead", "Tech Manager" and others are very stretchy. Companies solve different tasks, these roles have different responsibilities. Besides, various leadership styles, differences in personal skills, and much more are put on the picture. But isn't that what happens to the developers? Different companies, different tasks, again, different nature of tasks and responsibilities.
```
?

File: translations/README-en.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+We interviewed several dozen large and small companies, found out what they expect from Team Leads and how they interact with them. As a result, we were able to form a single model of basic teamwork competencies that covers all the cases studied.
+
+This model can be used in any way you like - to come up with your development plan, to create job descriptions in companies, to write a job description or to conduct interviews. Keep in mind it is likely you don't need all the branches of potential development - and this is fine.
+
+> 🚨 **Important:** Depending on the company, the Team Lead needs to work on different branches and responsibilities. We don't think that the goal should be to develop all of the areas in the roadmap. It's important to focus on the problems and needs of your workplace, first of all.
```
Comment (php-coder) [inline translations/README-en.md]:
```suggestion
> 🚨 **Important:** Depending on a company, the Team Lead needs to work on different branches and responsibilities. We don't think that the goal should be to develop all of the areas in the roadmap. It's important to focus on the problems and needs of your workplace, first of all.
```

File: translations/README-en.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+
+The work is still in progress, so feel free to publish an Issue, drop your pull request and join the discussion [in Telegram chat, RU](https://tlinks.run/tlbootcamp).
+
+# How to use it?
+## 🏫 For a company
+1. Download the [MindMap](roadmap.mm) with the full Team Lead model.
```
Comment (php-coder) [inline translations/README-en.md]:
```suggestion
1. Download the [MindMap](roadmap.mm) with the Team Lead model.
```

Comment (Gaket) [reply translations/README-en.md]: Here, the "full" word has its special meaning, I don't think it should be removed.

Comment (php-coder) [reply translations/README-en.md]: It's not clear what is special in this meaning (do we have a short Team Lead model?) but I don't mind to leave it as-is.

File: translations/README-en.md [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+This model can be used in any way you like - to come up with your development plan, to create job descriptions in companies, to write a job description or to conduct interviews. Keep in mind it is likely you don't need all the branches of potential development - and this is fine.
+
+> 🚨 **Important:** Depending on a company, the Team Lead needs to work on different branches and responsibilities. We don't think that the goal should be to develop all the areas of the roadmap. It's important to focus on the problems and needs of your workplace, first of all.
+
+The work is still in progress, so feel free to publish an Issue, drop your pull request and join the discussion [in Telegram chat, RU](https://tlinks.run/tlbootcamp).
+
+# How to use it?
+## 🏫 For a company
+1. Download the [MindMap](roadmap.mm) with the full Team Lead model.
+2. Go through all branches of the Team Lead responsibilities. Remove those that are not required by your company or are already being performed by someone else.
```
Comment (php-coder) [inline translations/README-en.md]:
```suggestion
2. Go through all the branches of the Team Lead responsibilities. Remove those that are not required by your company or are already being performed by someone else.
```
?

File: translations/README-en.md [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+3. Make a list of competencies that that separate your current profile and your target profile.
+4. Using our knowledge base, create a development plan for each of the competencies, which includes both theory and practical application.
+5. Show your development plan to your manager and ask for assistance.
+
+# Roadmap
+
+![Team Lead Map](../roadmap.png)
+
+The roadmap consists of two sections:
+- **Роли и обязанности.** A list of high-level work roles and more specific responsibilities.
```
Comment (php-coder) [inline translations/README-en.md]:
```suggestion
- **Roles and responsibilities.** A list of high-level work roles and more specific responsibilities.
```

File: translations/README-en.md [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
+- **Personal Skills.** Personal skills and qualities that are necessary for certain roles and responsibilities.
+
+Each of the map's leaves will eventually become a large entry in the knowledge base, containing a description, examples of behavior, and ways to develop the skill.
```
Comment (php-coder) [inline translations/README-en.md]:
```suggestion
Each of the map's leaves will eventually become a large entry in the knowledge base with a description, examples of behavior, and ways to develop the skill.
```
?

File: translations/README-en.md [hunk 7]
```diff
@@ -0,0 +1,218 @@
...
+
+## Download roadmap
+- [PNG](../roadmap.png)
+- [MindMap](../roadmap.mm)
+
+The easiest way to open a `mm` file is to visit [Freemind](https://sourceforge.net/projects/freemind/) (free and simple), [MindNode](https://mindnode.com/) (expensive and premium) or [MindMeister](https://mindmeister.com) (freemium and online). Other MindMap editors are also likely to work, but we haven't tested them.
```
Comment (php-coder) [inline translations/README-en.md]:
```suggestion
The easiest way to open a `mm` file is to use [Freemind](https://sourceforge.net/projects/freemind/) (free and simple), [MindNode](https://mindnode.com/) (expensive and premium) or [MindMeister](https://mindmeister.com) (freemium and online). Other MindMap editors are also likely to work, but we haven't tested them.
```

File: translations/README-en.md [hunk 8]
```diff
@@ -0,0 +1,218 @@
...
+
+## Examples of use
+We've prepared a few common Team Lead models. With their help, you can see how to work with the roadmap, or you can just bring them to your company as examples:
+- Playing coach in a scrum team: [MindMap](../examples/playing-coach.mm)
+- Technical Lead: [MindMap](../examples/techlead.mm)
+- Startup Team Lead: [MindMap](../examples/startup-teamlead.mm)
+
+# Work in progress
+The current roadmap is an alpha version that has a common teamwork structure and partially describes the first major role, Resource Manager. We are looking forward to your participation in the project, the process of contribution is described in detail in [`CONTRIBUTING.md`](CONTRIBUTING.md).
```
Comment (php-coder) [inline translations/README-en.md]:
```suggestion
The current roadmap is an alpha version that has a common teamwork structure and partially describes the first major role, Resource Manager. We are looking forward to your participation in the project. The contribution process is described in details in [`CONTRIBUTING.md`](CONTRIBUTING.md).
```
or
```suggestion
The current roadmap is an alpha version that has a common teamwork structure and partially describes the first major role, Resource Manager. We are looking forward to your participation in the project. The contribution process is described in depth in [`CONTRIBUTING.md`](CONTRIBUTING.md).
```

File: translations/README-en.md [hunk 9]
```diff
@@ -0,0 +1,218 @@
...
+        - PMBoK
+        - PDCA
+- Stakeholder
+
+## Personal Skills
+
+#### Self development
+- Ability to learn
+- Reflexion
+- Making habits.
```
Comment (php-coder) [inline translations/README-en.md]:
```suggestion
- Making habits
```

File: translations/README-en.md [hunk 10]
```diff
@@ -0,0 +1,218 @@
...
+
+# Work in progress
+The current roadmap is an alpha version that has a common teamwork structure and partially describes the first major role, Resource Manager. We are looking forward to your participation in the project, the process of contribution is described in detail in [`CONTRIBUTING.md`](CONTRIBUTING.md).
+
+# The knowledge base
+The base is gradually filled with Stas Tsyganov, Egor Tolstoy and the community. Our final goal is to describe each of the directions according to the following template:
+- Description (what we are talking about)
+- Why is a branch important
+- What happens if you don't work on it
+- Who can be delegated
```
Comment (php-coder) [inline translations/README-en.md]:
```suggestion
- To whom it can be delegated
```

File: translations/README-en.md [hunk 11]
```diff
@@ -0,0 +1,218 @@
...
+- Technical Lead: [MindMap](../examples/techlead.mm)
+- Startup Team Lead: [MindMap](../examples/startup-teamlead.mm)
+
+# Work in progress
+The current roadmap is an alpha version that has a common teamwork structure and partially describes the first major role, Resource Manager. We are looking forward to your participation in the project, the process of contribution is described in detail in [`CONTRIBUTING.md`](CONTRIBUTING.md).
+
+# The knowledge base
+The base is gradually filled with Stas Tsyganov, Egor Tolstoy and the community. Our final goal is to describe each of the directions according to the following template:
+- Description (what we are talking about)
+- Why is a branch important
```
Comment (php-coder) [inline translations/README-en.md]:
```suggestion
- Why this branch is important
```

File: translations/README-en.md (hunk 3 above)
Comment (Gaket) [reply translations/README-en.md]: Yep, we have subsets of the model, in "examples"

Comment (Gaket) [reply translations/README-en.md]: Let's see if there are some other opinions. As a variant, we can change "full" to "complete"

File: translations/README-en.md (hunk 4 above)
Comment (Gaket) [reply translations/README-en.md]: Redundant, imho. That's definitely not a grammatical error and I don't see a reason to add more words here.

File: translations/README-en.md (hunk 5 above)
Comment (Gaket) [reply translations/README-en.md]: Good catch

File: translations/README-en.md (hunk 6 above)
Comment (Gaket) [reply translations/README-en.md]: Not sure. Other opinions?

**Comment by Gaket** [comment]
Cool, thanks @php-coder for the corrections!

**Comment by etolstoy** [comment]
I've once maintained a repo with two versions – english and russian. The main problem was to keep both versions consistent. What process do you suggest to handle this issue in our case?

**Comment by Gaket** [comment]
I think that the best way is to have smth like a clearer versioning system. In the same time, it works as in CAP theorem - there are trade-offs everywhere. I don't think we need real-time consistency, eventual consistency should work in this case.

To facilitate the process of finding and translating the diff, we need to know how to calculate the diff. One way is to increment a version number of the main docs every time when an update happens. That's too cumbersome.

I suggest to add a commit hash of the original doc (in our case, let's promote Russian version to be the main one), and add that commit hash to the translated page, smth like "Translated version: fe73f23a0ff29514ef4b145740fe9fc729a235fa". 

Next time if anyone wants to update a translation, they need to look at the diff between the commit found with the latest hash in the translation and the current hash of the actual version.

If somebody wants to add a doc in non-Russian language, they should first find a person who can help in translation that doc to Russian, commit the Russian version and then commit the original with a commit hash of Russian. 

This way it will be a unidirectional system with a simple way of finding a diff to be translated.

File: translations/README-en.md [hunk 12]
```diff
@@ -0,0 +1,218 @@
+![Logotype](../logo.png)
+
+> Team Lead is a ❄️, because in each company they are unique and distinct.
+
+- [Roadmap](#roadmap)
+- [Telegram chat TL Bootcamp (RU)](https://tlinks.run/tlbootcamp)
```
Comment (etolstoy) [inline translations/README-en.md]: Давай спилим вообще, наверное. Если пойдет в массы, сделаем английскую версию.

File: translations/README-en.md [hunk 13]
```diff
@@ -0,0 +1,218 @@
...
+
+# What is it about?
+
+The concepts of "Team Lead", "Tech Lead", "Tech Manager" and others are very stretchy. Companies solve different tasks, these roles have different responsibilities. Besides, various leadership styles, differences in personal skills, and much more are put in the picture. But isn't that what happens to the developers? Different companies, different tasks, again, different nature of tasks and responsibilities.
+
+We interviewed several dozen large and small companies, found out what they expect from Team Leads and how they interact with them. As a result, we were able to form a single model of basic teamwork competencies that covers all the cases studied.
+
+This model can be used in any way you like - to come up with your development plan, to create job descriptions in companies, to write a job description or to conduct interviews. Keep in mind it is likely you don't need all the branches of potential development - and this is fine.
```
Comment (etolstoy) [inline translations/README-en.md]: В оригинале "должностные инструкции"/"описания вакансий". Здесь одно и то же выражение "job description", выглядит странно.

File: translations/README-en.md [hunk 14]
```diff
@@ -0,0 +1,218 @@
...
+
+> 🚨 **Important:** Depending on a company, the Team Lead needs to work on different branches and responsibilities. We don't think that the goal should be to develop all the areas of the roadmap. It's important to focus on the problems and needs of your workplace, first of all.
+
+The work is still in progress, so feel free to publish an Issue, drop your pull request and join the discussion [in Telegram chat, RU](https://tlinks.run/tlbootcamp).
```
Comment (etolstoy) [inline translations/README-en.md]: Тут тоже про чатик спилить лучше

File: translations/README-en.md [hunk 15]
```diff
@@ -0,0 +1,218 @@
...
+# How to use it?
+## 🏫 For a company
+1. Download the [MindMap](roadmap.mm) with the full Team Lead model.
+2. Go through all branches of the Team Lead responsibilities. Remove those that are not required by your company or are already being performed by someone else.
+3. Form from the resulting map the artifacts you need: a profile to hire, a description of role expectations, a development plan.
+4. Use our knowledge base to go deeper into any of the branches. For each of the branches we describe in detail its meaning, motivation to use, examples of good and bad behavior, ways of development in practice and theory.
+
+## 👩🏼‍💻 For yourself
+1. Download the [MindMap](roadmap.mm) with the full Team Lead model.
+2. Mark on it the competencies you already have and those you need for further growth within the company. To give you a hint, look at what your manager or colleagues are doing. If there are still difficulties, ask a question [in our chat room, RU](https://tlinks.run/tlbootcamp).
```
Comment (etolstoy) [inline translations/README-en.md]: Тоже чат

File: translations/README-en.md [hunk 16]
```diff
@@ -0,0 +1,218 @@
...
+
+# How to use it?
+## 🏫 For a company
+1. Download the [MindMap](roadmap.mm) with the full Team Lead model.
+2. Go through all branches of the Team Lead responsibilities. Remove those that are not required by your company or are already being performed by someone else.
+3. Form from the resulting map the artifacts you need: a profile to hire, a description of role expectations, a development plan.
+4. Use our knowledge base to go deeper into any of the branches. For each of the branches we describe in detail its meaning, motivation to use, examples of good and bad behavior, ways of development in practice and theory.
+
+## 👩🏼‍💻 For yourself
+1. Download the [MindMap](roadmap.mm) with the full Team Lead model.
```
Comment (etolstoy) [inline translations/README-en.md]: Нам надо либо сразу перевести майндмеп в этом PR, либо указать в описании, что он только на русском.

**Comment by etolstoy** [comment]
And @Gaket , let's try your system with a commit hash. Will you add its support?

File: translations/README-en.md (hunk 6 above)
Comment (AlexSkrypnyk) [reply translations/README-en.md]: @php-coder's suggestion makes more sense

File: translations/README-en.md (hunk 13 above)
Comment (AlexSkrypnyk) [reply translations/README-en.md]: "define duties and responsibilities"

File: translations/README-en.md (hunk 14 above)
Comment (AlexSkrypnyk) [reply translations/README-en.md]:
```
open a your pull request or join a discussion
```

**Comment by Gaket** [comment]
> And @Gaket , let's try your system with a commit hash. Will you add its support?

Yep, will do it today or tomorrow

File: translations/README-en.md (hunk 14 above)
Comment (Gaket) [reply translations/README-en.md]: fixed

**Comment by Gaket** [comment]
Fixed comments, added a draft pr with a suggested Translation process:
https://github.com/tlbootcamp/tlroadmap/pull/141

File: translations/README-en.md [hunk 17]
```diff
@@ -0,0 +1,220 @@
+![Logotype](../logo.png)
```
Comment (vvscode) [inline translations/README-en.md]: How about https://gitlocalize.com/ ? Keeping translation updated might be tricky

Comment (Gaket) [reply translations/README-en.md]: Looks interesting. Can you Add the same comment to https://github.com/tlbootcamp/tlroadmap/pull/141, to leave this pr for one page translation and discuss the overall process in the other one?

**Comment by etolstoy** [comment]
От меня аппрув, Стас еще посмотрит и сольем. В #141 давайте дальнейшие шаги тогда обсуждать.

**Pull request merged**